app/assets/javascripts/application.js

```diff
@@ -20,10 +20,19 @@ var Hours = Hours || {
   }
 };
 
+function setLanguage() {
+  if (I18n[$('body').data('language')] === undefined) {
```

Comment:
I guess this could be written as `return I18n[$('body').data('language') || I18n['en']`, and a more appropriate function name would probably be `getLanguage()`, since we're not setting anything. I just noticed this after merging, I can open a follow-up PR to fix it, or if anyone else get's around it that would be great.

Reply:
I'll send you this change.